File: Lesson7 String Algorithms/Problem1/Solutions/Solution_brute_force.cpp
#include <iostream>

using namespace std;

int main()
{
   string s; cin>>s;
   bool found = false;
   int len = s.length();
   for(int i = 0; i < len - 2; i++){
       if(s[i] == 'C' && s[i + 1] == 'C' && s[i + 2] == 'C'){
           found = true;
           break;
       }
   }
   if(found){
       cout<<"NO"<<endl;
   }
   else{
       cout<<"YES"<<endl;
   }
   return 0;
}

File: Lesson7 String Algorithms/Problem1/Solutions/Solution_brute_force.java
import java.util.Scanner;
public class Solution_brute_force{
     public static void main(String []args){
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        boolean found = false;
        for(int i = 0; i < s.length() - 2; i++){
            if(s.charAt(i) == 'C' && s.charAt(i + 1) == 'C' && s.charAt(i + 2) == 'C'){
                found = true;
                break;
            }
        }
        if(found){
            System.out.println("NO");
        }
        else {
            System.out.println("YES");
        }
        sc.close();
     }
}
